packages/docs/src/page-configs/ui-elements/modal/modal-api.ts

```diff
@@ -0,0 +1,86 @@
+import { PageGenerationHelper } from '../../../helpers/DocsHelper'
+import { ModalOptions } from '../../../../../ui/src/components/va-modal/types'
+
+export const methodsApi = (block: PageGenerationHelper) => block.table(
+  [
+    { title: 'Method name', type: 'markdown' },
+    { title: 'Api Type', type: 'markdown' },
+    { title: 'Description', type: 'markdown' },
+  ],
+  [
+    ['init(options: string | ModalOptions)', 'Options | Composition', 'Creates new modal instance. Returns modal instance VNode'],
+  ],
+)
+
+const optionsTableData: [keyof ModalOptions, string][] = [
```

Comment:
Better have object typeof ModalOptions here. We probably will forget to add some props here...

```ts
const o: Record<keyof ModalOptions, string>
```

Will it throw error if we forget to add `title` key here?